hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -87,30 +89,30 @@ public void testSchemaConvertArray() throws IOException {
         .named("ArrayOfInts");
 
     String schemaString = HiveSchemaUtil.generateSchemaString(schema);
-    assertEquals("`int_list` ARRAY< int>", schemaString);
+    assertEquals("int_list ARRAY<int>", schemaString);
```

Comment:
Can you add test coverage for DLASchemaUtil#generateSchemaString (and other methods in DLASchemaUtil?)

Reply:
Now I have added the TestDLASyncTool class to test the methods of the DLASchemaUtil class.please review.thanks.